src/types/policies/ITC.jl

```diff
@@ -4,11 +4,17 @@
 
 Investment Tax Credit - A tax incentive that is a percentage of capital cost given to generators that meet the qualifications. 
 
-# Keyword Arguments
-
+### Keyword Arguments:
 * `name`: policy name
 * `values`: the credit level, stored as an OrderedDict with year and value `(:y2020=>0.3)`.  Credit level refers to the percentage of the capex that will be rebated to the investor
 * `gen_filters`: filters for qualifying generators, stored as an OrderedDict with gen table columns and values (`:emis_co2=>"<=0.1"` for co2 emission rate less than or equal to 0.1)
+
+### Table Column Added: 
+* `(:gen, :pol.name)` - the investment tax credit value for the policy 
+
+### Results Formula:
+* `(:gen, :total_result_sym)` - the cost of the policy, as seen by the objective, not used for government spending welfare
```

Comment:
`total_result_sym` is the name of the julia variable, could you find what goes into it?  I.e. `:<name>_result` or whatever it is.